index.rst

```diff
@@ -38,14 +38,21 @@ It is worth noting that:
 #. The manager considers each file in the handoff buffer to be a new file ready
    for transfer. Hence, for the manager to operate correctly, writing files to
    the handoff buffer must be an atomic operation.
-#. Similarly to rsync, the manager replicates the directory structure between
-   buffers. For example, the file in the handoff buffer located at
-   foo/bar/baz.fits  will be written to exactly the same location in the
+#. Similarly to `rsync`_, the manager replicates the directory structure
+   between buffers. For example, the file in the handoff buffer located at
+   ``foo/bar/baz.fits`` will be written to exactly the same location in the
    endpoint buffer.
-#. The current transfer mechanism is ``scp``, so the handoff manager must first
+#. File transfer mechanism can be easily changed.
```

Comment:
Is this via the config file?  Is there a couple words to limit what mechanisms can be used?  For example "(must be able to write a command line containing a placeholder for the source and destination of individual file transfers)"

Reply:
You're right, it's a bit unclear. I reword/expand this paragraph according to your suggestions.